projects/igniteui-angular/src/lib/combo/combo.common.ts

```diff
@@ -1187,10 +1187,14 @@ export abstract class IgxComboBaseDirective extends DisplayDensityBase implement
     };
 
     protected manageRequiredAsterisk(): void {
-        if (this.ngControl && this.ngControl.control.validator) {
-            // Run the validation with empty object to check if required is enabled.
-            const error = this.ngControl.control.validator({} as AbstractControl);
-            this.inputGroup.isRequired = error && error.required;
+        if (this.ngControl) {
```

Comment:
We may want to add a comment in the following form - `<initials> <date> <Issue Number|Issue Link> <Comment for the change>`